opm/simulators/wells/BlackoilWellModelGeneric.hpp

```diff
@@ -105,6 +105,9 @@ class BlackoilWellModelGeneric
     bool wellsActive() const;
     bool hasWell(const std::string& wname) const;
 
+    // return true if well exists and is not shut
```

Comment:
This should be more specific, in light of the discussion in https://github.com/OPM/opm-simulators/pull/5849, I assume this is just on this process? If so, we should call the method something with Local. But I am not sure exactly what role this plays compared to the other has...Wells() methods?

Reply:
I'll use `hasOpenLocalWell` from https://github.com/OPM/opm-simulators/pull/5849 once that is merged.